umbral-pre-wasm/.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
 package-lock.json
+pkg
```

Comment:
It seems that despite that there's still `pkg` folder committed with `LICENSE`, `package.json` and `README.md` in it. Added by mistake?